lib/assessment.js

```diff
@@ -20,7 +24,10 @@ var sum = function sum() {
 // Write a function that returns an array of the specified length
 // with all the elements set to defaultValue
 var newArray = function newArray(length, defaultValue) {
+  var length = this.length;
```

Comment:
Since this function is not called as a method, `this` will resolve either to the global object or `undefined`.